.eslintrc.json

```diff
@@ -1,22 +1,18 @@
 {
-    "env": {
-        "browser": true,
-        "es2021": true
+  "env": {
+    "browser": true,
+    "es2021": true
+  },
+  "extends": ["plugin:react/recommended", "airbnb"],
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
     },
-    "extends": [
-        "plugin:react/recommended",
-        "airbnb"
-    ],
-    "parserOptions": {
-        "ecmaFeatures": {
-            "jsx": true
-        },
-        "ecmaVersion": 12,
-        "sourceType": "module"
-    },
-    "plugins": [
-        "react"
-    ],
-    "rules": {
-    }
+    "ecmaVersion": 12,
+    "sourceType": "module"
+  },
+  "plugins": ["react"],
+  "rules": {
+    "linebreak-style": 0
```

Comment:
a to potrzebujemy?

Reply:
Tak ja to w windowsie potrzebuję. Jak zrobiłem nowego brancha to wrócił mi błąd _**"Expected linebreaks to be 'LF' but found 'CRLF'"**_, gdzieś na stacku znalazłem że to powinno pomóc i pomogło.